cpp/include/games/nim/Game.hpp

```diff
@@ -0,0 +1,157 @@
+
+#pragma once
+
+#include <array>
+#include <cstdint>
+#include <functional>
+#include <sstream>
+#include <string>
+
+#include <boost/functional/hash.hpp>
+#include <torch/torch.h>
+
+#include <core/BasicTypes.hpp>
+#include <core/concepts/Game.hpp>
+#include <core/GameLog.hpp>
+#include <core/GameTypes.hpp>
+#include <core/SimpleStateHistory.hpp>
+#include <core/TrainingTargets.hpp>
+#include <core/WinShareResults.hpp>
+#include <util/EigenUtil.hpp>
+#include <util/FiniteGroups.hpp>
+#include <util/MetaProgramming.hpp>
+#include <games/nim/Constants.hpp>
+#include <core/TrivialSymmetries.hpp>
+
+namespace nim {
+
+struct Game {
+  struct Constants {
+    static constexpr int kNumPlayers = nim::kNumPlayers;
+    static constexpr int kNumActions = nim::kMaxStonesToTake;
+    static constexpr int kMaxBranchingFactor = nim::kMaxStonesToTake;
+    static constexpr int kNumPreviousStatesToEncode = 0;
+    static constexpr float kOpeningLength = 0.1;  // not applicable to Nim
```

Comment:
kOpeningLength is used to provide default configurations for some MCTS-parameters. So it's actually worth providing a reasonable value here. I'd say something like 3 might be reasonable.